rotary/rules/rotary.smk

```diff
@@ -312,16 +367,180 @@ rule qc_long_length_stats:
         length_stats.to_csv(output[0], sep='\t', header=None, index=True)
 
 
+rule finalize_qc_long:
+    input:
+        "{sample}/qc/long/{sample}_nanopore_qc.fastq.gz"
+    output:
+        "{sample}/qc/{sample}_QC_long.fastq.gz"
```

Comment:
@jmtsuji This might be a bit nitpicky, but I was wondering if there was a reason why QC is capitalized? All the other files with QC in their file names have the 'qc' lower case.

Reply:
Thanks for noticing this! I've converted all to lowercase.